cloud_cleaner/resources/fip.py

```diff
@@ -36,10 +36,10 @@ def register(self, config):
                                       dest='static_subnet',
                                       help=_desc)
 
-    def process(self):
-        shade = self._get_shade()
+    def process(self, deletion):
```

Comment:
I don't know if I'm missing it, but I don't see "deletion" being used in this method anywhere.